### ISSUE TITLE
Investigate and fix deprecation warnings

### ISSUE THREAD

**Issue by lxp** [cfv-project/cfv#21]
Some deprecated modules and functions are used. We have to investigate those and move them to a proper supported module/function.

Python provides the `-W error` argument to ease finding such issues. For example:

```sh
$ python3 -W error test/test.py
Traceback (most recent call last):
  File "test/test.py", line 48, in <module>
    import cfvtest
  File "/[...]/cfv/test/cfvtest.py", line 26, in <module>
    import imp
  File "/usr/lib/python3.6/imp.py", line 33, in <module>
    DeprecationWarning, stacklevel=2)
DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
```

Once fixed, we should also add these Python flags in our CI pipeline.

**Comment by lxp** [comment]
`assertEquals` deprecation was fixed with pull request #43 
`imp` module deprecation is discussed in issue #44

**Comment by lxp** [comment]
Another deprecation warning appeared:
#57
